source/module_io/numerical_basis.cpp

```diff
@@ -139,12 +139,11 @@ void Numerical_Basis::output_overlap(const psi::Psi<std::complex<double>>& psi,
         }
         Parallel_Reduce::reduce_pool(overlap_V.c, overlap_V.nr * overlap_V.nc);		// Peize Lin add 2020.04.23
     #endif
+        if(ofs.good()) this->output_info(ofs, bessel_basis, kv); else assert(false);
```

Comment:
"assert" will stpp the program without printing any useful information, why not use "WARNING_QUIT("...","...")" ?

Reply:
Thanks for pointing out this, please see the modified code